thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/MetricConfigResource.java

```diff
@@ -77,7 +77,9 @@ private void populateMetricConfig(MetricConfigDTO metricConfigDTO, String datase
     // optional ones
     metricConfigDTO.setCellSizeExpression(cellSizeExpression);
     metricConfigDTO.setInverseMetric(inverseMetric);
-    metricConfigDTO.setRollupThreshold(rollupThreshold);
+    if (rollupThreshold != null) {
```

Comment:
you can set the defaultValue in the java code. PathParam can have the default value as well.